tests/pipelines/eks/awscli-eks-cl2-load.yaml

```diff
@@ -11,8 +11,10 @@ spec:
   - name: cluster-name
   - name: endpoint
   - name: servicerole
+    default: ""
```

Comment:
you can remove this at pipeline level, given it's already a default at task level IIUC.

Reply:
removing this from here will not allow us to trigger runs without empty params

Reply:
as discussed offline removed this. The user needs to explicitly pass role names for these pipelines.